content/6_Advanced/DP_More.problems.json

```diff
@@ -1,5 +1,43 @@
 {
   "MODULE_ID": "dp-more",
+  "knuth_optimization": [
+    {
+      "uniqueId": "spoj-breaking-strings",
+      "name": "Breaking Strings",
+      "url": "https://www.spoj.com/problems/BRKSTRNG/",
+      "source": "SPOJ",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": [],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "uva-gain-battle-power",
+      "name": "Gain Battle Power",
+      "url": "https://onlinejudge.org/external/128/12836.pdf",
+      "source": "UVa Online Judge",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": [],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "optimal-binary-search-tree",
+      "name": "Optimal Binary Search Tree",
+      "url": "https://onlinejudge.org/external/103/10304.pdf",
```

Comment:
where 2 submit

Reply:
Updated the links. The site is very old though so the user experience might not be that great. I also added a solution on geeksforgeeks to that problem.

Sorry for the messy commit history. This is my first pull request on this repository, so I'm still learning.

Reply:
yee dw